cmd/lotus-shed/dealtracker.go

```diff
@@ -0,0 +1,265 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"net"
+	"net/http"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/lotus/api"
+	lcli "github.com/filecoin-project/lotus/cli"
+	"github.com/ipfs/go-cid"
+	"github.com/urfave/cli/v2"
+)
+
+type dealStatsServer struct {
+	api api.FullNode
+}
+
+var filteredClients map[address.Address]bool
+
+func init() {
+	filteredClients = make(map[address.Address]bool)
+	for _, a := range []string{"t0112", "t0113", "t0114", "t010089"} {
```

Comment:
And is there some way that we can make this configurable? E.g. if we want to add more addresses to filter out of the search down the road?

Reply:
yeah, I can make it into an environment variable

Reply:
Awesome!